dart/lib/src/protocol/sentry_stack_frame.dart

```diff
@@ -0,0 +1,106 @@
+class SentryStackFrame {
+  static final SentryStackFrame asynchronousGapFrameJson = SentryStackFrame()
+    ..absPath = '<asynchronous suspension>';
+
+  List<String> preContext;
+  List<String> postContext;
+  Map<String, String> vars;
+  List<int> framesOmitted;
+  String filename;
+  String function;
+  String module;
+  String lineno;
+  String colno;
```

Comment:
some data types are wrong, `lineno` and `colno` are int, for example, I'd double-check all new additions to this PR otherwise we'll probably see backend errors

Reply:
✅